### PR TITLE
Run3-ft32 Add files for zero material scenario

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Add files for zero material scenario of the tracker after removing tracker specific materials from materials.xml

#### PR validation:

Tested using specific cfg's in the test area of Geometry/TrackerCommonData

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special;

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/TrackerCommonData/python/cmsExtendedGeometry2021ZeroMaterialXML_cfi.py [hunk 1]
```diff
@@ -0,0 +1,380 @@
+import FWCore.ParameterSet.Config as cms
```
Comment (cvuosalo) [inline Geometry/TrackerCommonData/python/cmsExtendedGeometry2021ZeroMaterialXML_cfi.py:1]: Is this file generated? If not, why not? You added the capability to do so in #31082.

Comment (cvuosalo) [reply Geometry/TrackerCommonData/python/cmsExtendedGeometry2021ZeroMaterialXML_cfi.py:1]: Or maybe the incorrect comment in the other PR has me confused.

**Comment by cvuosalo** [comment]
+1

**Comment by silviodonato** [comment]
+1

**Pull request merged**